packages/lib-panoptes-js/panoptes.js

```diff
@@ -0,0 +1,67 @@
+const superagent = require('superagent');
+const config = require('./config');
+
+function handleErrors(error) {
+  if (console) console.error(error);
+
+  return error;
+}
+
+// TODO: Write auth client
```

Comment:
Instead of including an auth client, it could support passing in an auth instance (whatever that ends up looking like), or a Bearer token as an argument - this feels like it should remain a transport library

Reply:
That's a good idea. I was thinking about this yesterday that since we're going the route of server side rendering, what do you think about doing oauth server side as well? Maybe using a library like this? https://github.com/ciaranj/node-oauth